### ISSUE TITLE
DFS mount in image 5.4.0-1056-azure

### ISSUE THREAD

**Issue by mschuurmans** [Azure/AKS#2588]
When i'm mounting a DFS server using cifs im getting the error

```
CFS VFS: BAD_NETWORK_NAME: \\<server>\path
```
using the command:
```
mount -t cifs //<server>/path /mnt/dfs/ -o username=<username>,password=<password>,vers=3.0
```
The kernel version on the AKS node is 5.4.0-1056-azure

After some digging i found maybe some related issues https://www.spinics.net/lists/linux-cifs/msg18290.html and https://bugs.launchpad.net/ubuntu/+source/linux/+bug/1854887

The second is a bug i experienced when testing before but can now not reproduce because the initial mount keeps failing. If i mount the same server (with the same command) on my local laptop running linux 5.14.0-1-amd (Debian testing) the dfs mounting works just fine.

**Anything else we need to know?**:

**Environment**:
- Kubernetes version (use `kubectl version`): 1:21.2
- Size of cluster: 6 nodes
- General description of workloads in the cluster: Http microservices
- Others:

**Comment by andyzhangx** [comment]
same issue as https://github.com/Azure/AKS/issues/2292, @RenaShahMSFT do you know which kernel fixed this issue? AKS node kernel is now `Linux version 5.4.0-1056-azure`

**Comment by olsenme** [comment]
@RenaShahMSFT

**Comment by VybavaRamadoss** [comment]
> @andyzhangx, @edreed, @ZeroMagic, @VybavaRamadoss would you be able to assist?
> 
> Issue Details

Looks like the fix is in Kernel version 5.4.112. Can we upgrade the AKS nodes to this version? Per https://github.com/Azure/AKS/issues/2292 the ask is to avoid versions between 5.4.0-1043 to 5.4.112.

**Comment by andyzhangx** [comment]
latest AKS node linux kernal version is `5.4.0-1083-azure`
